libc-top-half/musl/src/thread/pthread_attr_setschedparam.c

```diff
@@ -2,6 +2,11 @@
 
 int pthread_attr_setschedparam(pthread_attr_t *restrict a, const struct sched_param *restrict param)
 {
+#ifdef __wasilibc_unmodified_upstream
 	a->_a_prio = param->sched_priority;
 	return 0;
+#else
+	if (param->sched_priority == 0) return 0;
+	return ENOTSUP;
```

Comment:
Can we flip this so the error case is first and `return 0` is the last line?  (then that last line can also go outside the if/else)

Reply:
Done